source/components/molecules/GroupedList/GroupedList.tsx

```diff
@@ -66,11 +80,18 @@ const GroupedList: React.FC<Props> = ({
         property: LayoutAnimation.Properties.opacity,
       },
     });
+    // Sort whenever the summarylist is closed for editing
+    // i.e the 'Close' button is tapped
+    if (editable && sortCallback) {
+      sortCallback(children.map((element) => element.props.item));
+    }
     setEditable(!editable);
   };
 
   categories.forEach((cat) => {
-    const catItems = children.filter((item) => item.props.category === cat.category);
+    const catItems = children.filter(
```

Comment:
Cat? 🐱. Prefer to not use abbreviations when possible.

Reply:
Have not changed that but fixed it anyways

Reply:
Ah It was just formatting, but might as well cleanup 👍.